charts/kafka/values.yaml

```diff
@@ -1,3 +1,5 @@
+# Copyright 2020 Crown Copyright
+#
```

Comment:
Can't add crown copyright notice as this was not our original file

Reply:
This was by request from @ac74475 as the file was reasonably changed.
The license remains the same and it still includes the "Originally based on ...." link
At what percentage or total change do you go from "Original license" to "Crown copyright"?

Reply:
So the Apache 2 licence includes the following clause:
'You may add Your own copyright statement to Your modifications'
Therefore I believe we have done the correct thing and no changes are required for this comment